src/modules/file-upload-input.marko

```diff
@@ -0,0 +1,111 @@
+<div id="file-upload-input">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>
```

Comment:
I think we can reduce & remove a lot of this introductory content. We should only call out classes that require some special attention, otherwise we just let the developer refer to the example code.

<img width="671" alt="Screenshot 2024-09-10 at 10 55 02 AM" src="https://github.com/user-attachments/assets/19c34478-b363-4ad7-9d05-044c92d9dfba">